### PR TITLE
refactor(streaming): pass order_col_indices and order_types in minput

### PR DESCRIPTION
**Pull request by KeXiangWang**
I hereby agree to the terms of the [RisingWave Labs, Inc. Contributor License Agreement](https://gist.github.com/TennyZhuang/f00be7f16996ea48effb049aa7be4d66#file-rw_cla).

## What's changed and what's your intention?

Now `order by columns` of `MaterializedInputState` are generated on both frontend and compute nodes, which are deplicated. After this PR, it will be only generated on frontend and passed to computenodes. 

Related issue/PR: https://github.com/risingwavelabs/risingwave/issues/13044 and https://github.com/risingwavelabs/risingwave/pull/13015/

## Checklist

- [ ] I have written necessary rustdoc comments
- [ ] I have added necessary unit tests and integration tests
- [ ] I have added fuzzing tests or opened an issue to track them. (Optional, recommended for new SQL features #7934).
- [ ] My PR contains breaking changes. (If it deprecates some features, please create a tracking issue to remove them in the future).
- [x] All checks passed in `./risedev check` (or alias, `./risedev c`)
- [ ] My PR changes performance-critical code. (Please run macro/micro-benchmarks and show the results.)

- [ ] My PR contains critical fixes that are necessary to be merged into the latest release. (Please check out the [details](https://github.com/risingwavelabs/risingwave/blob/main/CONTRIBUTING.md))

## Documentation

- [ ] My PR needs documentation updates. (Please use the **Release note** section below to summarize the impact on users)

### CODE REVIEW

**Review by kwannoel — APPROVED** [review]

File: src/stream/src/executor/aggregation/minput.rs [hunk 1]
```diff
@@ -346,10 +310,14 @@ mod tests {
         )
         .await;
 
+        let order_columns = vec![
+            ColumnOrder::new(2, OrderType::ascending()), // min(c)
```
Comment (stdrc) [inline src/stream/src/executor/aggregation/minput.rs:314]:
```suggestion
            ColumnOrder::new(2, OrderType::ascending()), // c ASC for AggKind::Min
```

File: src/stream/src/executor/aggregation/minput.rs [hunk 2]
```diff
@@ -552,20 +522,28 @@ mod tests {
...
-            &input_pk_indices,
+            &order_columns_1,
             &mapping_1,
             usize::MAX,
             &input_schema,
         )
         .unwrap();
 
+        let order_columns_2 = vec![
+            ColumnOrder::new(1, OrderType::descending()), // b DESC for AggKind::Min
```
Comment (stdrc) [inline src/stream/src/executor/aggregation/minput.rs:540]:
```suggestion
            ColumnOrder::new(1, OrderType::descending()), // b DESC for AggKind::Max
```

**Review by stdrc — APPROVED** [review]
LGTM!

File: src/stream/src/executor/aggregation/minput.rs (hunk 1 above)
Comment (KeXiangWang) [reply src/stream/src/executor/aggregation/minput.rs]: updated

**Pull request merged**